test/evaluate.js

```diff
@@ -16,7 +17,7 @@ test('xdm (evaluate)', async function (t) {
 
   t.throws(
     function () {
-      // @ts-ignore runtime.js does not have a typing
+      // @ts-ignore Expected.
       evaluateSync('a', {Fragment: runtime.Fragment})
```

Comment:
is there a better way in tests to expect a type error?

Reply:
Since we're using TS 4, we could use https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments